src/github.com/mozilla-services/pushgo/simplepush/worker.go

```diff
@@ -756,23 +773,28 @@ func (self *WorkerWS) Ping(sock *PushWS, header *RequestHeader, _ []byte) (err e
 	}
 	self.lastPing = now
 	if self.app.pushLongPongs {
-		sock.Socket.WriteJSON(PingReply{header.Type, 200})
+		self.WriteJSON(PingReply{header.Type, 200})
 	} else {
-		sock.Socket.WriteText("{}")
+		self.WriteText("{}")
 	}
 	self.metrics.Increment("updates.client.ping")
 	return nil
 }
 
 // TESTING func, purge associated records for this UAID
-func (self *WorkerWS) Purge(sock *PushWS, _ *RequestHeader, _ []byte) (err error) {
+func (self *WorkerWS) Purge(header *RequestHeader, _ []byte) error {
 	/*
-	   // If needed...
-	   sock.Scmd <- PushCommand{Command: PURGE,
-	       Arguments:JsMap{"uaid": sock.UAID()}}
-	   result := <-sock.Scmd
+		// If needed...
+		uaid := self.UAID()
+		if uaid == "" {
+			return ErrInvalidCommand
+		}
+		err := self.store.DropAll(uaid)
+		status, _ := ErrToStatus(err)
+		self.WriteJSON(PingReply{header.Type, status})
+		return nil
 	*/
-	sock.Socket.WriteText("{}")
+	self.WriteText("{}")
 	return nil
```

Comment:
This was the Ping handler, back when we had a discrete handler for it. I think ping handling is currently done inline in the sniffer, so we may be able to drop this.